### DISCUSSION TITLE
Regex replacement while building the site

### DISCUSSION THREAD

**Discussion by ayhanfuat** [observablehq/framework#1137] [Q&A] [answered]
I am migrating a site that was previously being built with Python to Framework❤️. We had an extra step before building that site that would search for occurrences of a pattern (something like `PAT-123`) in markdown and replace it with a link (something like `[PAT-123](https://issue-tracker.com/PAT-123)`). Is there any built-in support for this or do I need to write/use a markdown-it plugin?

**Comment by mbostock** [comment, marked as answer]
There is no built-in support for this but I imagine you could do this with a markdown-it plugin. We could also add a config hook for rewriting HTML (post-Markdown-rendering).
